.github/workflows/ci.yml

```diff
@@ -77,9 +77,13 @@ jobs:
         targets: ${{ matrix.platform.target }}
         components: clippy, rust-src
 
+    - name: Install libwayland
+      if: (matrix.platform.os == 'ubuntu-latest')
+      run: sudo apt-get update && sudo apt-get install libwayland-dev
+
     - name: Install GCC Multilib
       if: (matrix.platform.os == 'ubuntu-latest') && contains(matrix.platform.target, 'i686')
-      run: sudo apt-get update && sudo apt-get install gcc-multilib
```

Comment:
Huh?

Reply:
The way Github chose to display the diff is interesting, but this line now runs just `apt-get install` and not `apt-get update`, since the previous step will already have done an `apt-get update`.

Reply:
Alright, that makes sense to me